storytime-block.php

```diff
@@ -64,14 +64,18 @@ function storytime_panel_render_callback( $attributes, $content ) {
 
 		$obj['panel_markup'] = $content;
 
-		// // Set up button data and base obj
-		// $panel_button_data   = $attributes['buttons'];
-		// $panel_button_struct = $obj['panel_buttons'][0];
+		// Get a copy of the button obj
+		$panel_button_struct = $obj['panel_buttons'][0];
```

Comment:
We need a lass manual way of plugging in the data...perhaps an object from Gutenberg containing real data could be written to the JSON? Then we wouldn't need this in PHP